client/repo.c

```diff
@@ -598,9 +598,8 @@ TDNFGetRepoMD(
     if(metalink)
     {
         dwError = TDNFAllocateStringPrintf(&pszRepoMetalink,
-                                          "%s/%s",
-                                          pRepoData->pszMetaLink,
-                                          TDNF_REPO_METALINK_FILE_PATH);
+                                          "%s",
+                                          pRepoData->pszMetaLink);
```

Comment:
Not sure about it, what happens to the previously created "repodata/photon.metalink" file?

Reply:
When we refresh we will anyways remove the while repodata folder which includes this(this was done with last PR https://github.com/vmware/tdnf/pull/134 )